AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -202,6 +204,12 @@ - (instancetype)initWithFrame:(CGRect)frame collectionViewLayout:(UICollectionVi
   // and should not trigger a relayout.
   _ignoreMaxSizeChange = CGSizeEqualToSize(_maxSizeForNodesConstrainedSize, CGSizeZero);
   
+  // Register the default layout inspector delegate for flow layouts, custom layouts
+  // will need to roll their own ASCollectionViewLayoutInspecting implementation.
+  _layoutDelegate = [self flowLayoutInspector];
```

Comment:
So that we can do more final reviews, can you prioritize refactoring this to be how you'd suggest landing it?  Naturally this doesn't make much sense for collections created without a flow layout; we should have some mechanism to ensure this is only done for collections with a flow layout.